Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/storage/internal/RiskResultDatabase.kt

```diff
@@ -81,11 +82,12 @@ abstract class RiskResultDatabase : RoomDatabase() {
 
     class Factory @Inject constructor(@AppContext private val context: Context) {
 
-        fun create(): RiskResultDatabase {
+        fun create(databaseName: String = DATABASE_NAME): RiskResultDatabase {
             Timber.d("Instantiating risk result database.")
             return Room
-                .databaseBuilder(context, RiskResultDatabase::class.java, DATABASE_NAME)
-                .fallbackToDestructiveMigrationFrom()
+                .databaseBuilder(context, RiskResultDatabase::class.java, databaseName)
+                .addMigrations(RiskResultDatabaseMigrations.MIGRATION_1_2)
+                .fallbackToDestructiveMigrationFrom(3)
```

Comment:
How this will behave as I did not see a schema with version 3? 
would `fallbackToDestructiveMigration()` be an option in this case?

Reply:
I've removed it as it was not needed here, thx!